### ISSUE TITLE
Technical Information

### ISSUE THREAD

**Issue by RJGhugo** [photoprism/photoprism#3755]
Beside "Moments" or "Labels" there would be nice to have a category that show all technical available informations (e.g. aperture, time, ISO).

**Comment by graciousgrey** [comment]
I think smart albums should solve this once they are implemented?

- https://github.com/photoprism/photoprism/issues/1431

**Comment by RJGhugo** [comment]
Thanks.

Smart albums could solve this in future, when all metadata (exif & iptc) will be available/imported...

**Comment by graciousgrey** [comment]
We already support and read a lot of metadata. Are you missing some technical fields? 
Iso, aperture, F-number etc. should all be read, see: https://www.photoprism.app/kb/metadata

With the upcoming release you will also be able to search for iso, focal length and aperture ranges:
- https://github.com/photoprism/photoprism/issues/3818

What is still missing are fields related to hierarchical tags, since we need to implement them in PhotoPrism before we can read the corresponding metadata fields.

**Comment by lastzero** [comment]
I'll go ahead and close this feature request as resolved, as it looks like the additional information is visible in the cards view when using our latest release! 👍

Related Issues:

- https://github.com/photoprism/photoprism/issues/3168#screenshots
- https://github.com/photoprism/photoprism/issues/3816
- https://github.com/photoprism/photoprism/issues/2040

**Comment by lastzero** [comment]
If this request was for filtering/organizing by camera/lens information, then search filters (which you can bookmark) and smart albums (i.e. more advanced bookmarks, once implemented) will provide that functionality.
